OpenEdXMobile/res/layout/fragment_whats_new.xml

```diff
@@ -3,12 +3,14 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <LinearLayout
+        android:id="@+id/ll_root"
```

Comment:
need id for title on all screens

Reply:
following story has been created for that
https://openedx.atlassian.net/browse/LEARNER-4408